.babelrc

```diff
@@ -1,5 +1,8 @@
 {
   "presets": ["react", "es2015", "stage-0"],
+  "plugins": [
+    'transform-runtime'
```

Comment:
This is a node module called `babel-plugin-transform-runtime` which is loaded by babel on its own, same with the presets.